src/webview/android/kotlin/TauriActivity.kt

```diff
@@ -0,0 +1,71 @@
+package {{app-domain-reversed}}.{{app-name-snake-case}}
```

Comment:
should this one be generated by tao?

Reply:
iirc, tao generates functions using `android_binding!` macro, which takes the app_domain and app_name as input.